# Based on a patch by Simon McVittie <smcv@debian.org>:
# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=847292#16

# Stops proot from segfaulting by disabling seccomp.
# https://github.com/proot-me/proot/issues/106

--- a/src/GNUmakefile
+++ b/src/GNUmakefile
@@ -132,7 +132,7 @@ ifneq ($(and $(HAS_SWIG),$(HAS_PYTHON_CONFIG)),)
        CHECK_PYTHON_EXTENSION = /bin/echo -e "\#define HAVE_PYTHON_EXTENSION"
 endif

-CHECK_FEATURES = process_vm seccomp_filter
+CHECK_FEATURES = process_vm
 CHECK_PROGRAMS = $(foreach feature,$(CHECK_FEATURES),.check_$(feature))
 CHECK_OBJECTS  = $(foreach feature,$(CHECK_FEATURES),.check_$(feature).o)
 CHECK_RESULTS  = $(foreach feature,$(CHECK_FEATURES),.check_$(feature).res)
